pkg/deploy/controller/deploymentconfig/deploymentconfig_controller.go

```diff
@@ -125,6 +148,15 @@ func (c *DeploymentConfigController) Handle(config *deployapi.DeploymentConfig)
 				if err != nil {
 					return err
 				}
+				// We need to make sure we own that RC or adopt it if possible
+				isOurs, err := cm.ClaimReplicationController(rc)
```

Comment:
> Now I am thinking that we never supported adopting RCs for DCs, before owner refs, so I am wondering if we are going to break users that have fiddled with RC labels. Maybe warn in the release notes since we are departing from the old way of selecting RCs.

Nevermind this one, rechecked and you merely add owner refs on top - should be fine.